README.md

```diff
@@ -175,8 +175,8 @@ an SD card. See [Using an SD card as storage](#using-an-sd-card-as-storage) for
 #### TCP Socket / IPC Socket
 
 To be able to connect remotely to the docker daemon on the device, `TCP Socket` needs to be selected.
-`IPC Socket` needs to be selected for containers running on the device to be able to communicate with each other.
-At least one of the sockets needs to be selected for the application to start dockerd.
+`IPC Socket` needs to be selected for containers running on the device to be able to communicate with
+each other. At least one of the sockets needs to be selected for the application to start dockerd.
```

Comment:
This could cause unnecessary conflicts between the repos. If lines are too long, it's better if we make a line-breaking delivery to Docker ACAP repo instead.

Reply:
done